pkg/sink/sink.go

```diff
@@ -91,17 +95,40 @@ type Response struct {
 	ErrorMessage string `json:"errorMessage,omitempty"`
 }
 
+func (r Sink) emitEvents(recorder record.EventRecorder, el *triggersv1.EventListener, err error) {
+	afterCondition := el.Status.GetCondition(apis.ConditionReady)
+	if el.Annotations[events.EnableEventListenerEvents] == "true" {
```

Comment:
@dibyom emitting events only when user sets `enable-eventlistener-events: "true"`  annotation
LMK your opinion on this.
If it looks okay i will update same info in Doc as well
/cc @khrm

Reply:
Yeah that sounds good. This should be an option in the config-features.

Reply:
@dibyom do you mean we no need to take from eventlistener annotation instead we should depend on `config-features` cm ?

Reply:
my bad. I missed that this was an annotation. Just an annotation is fine for now as long as we document it.